pytorch/2.1-a-first-look-at-a-neural-network.ipynb

```diff
@@ -0,0 +1,547 @@
+    "\n",
+    "test_loader = torch.utils.data.DataLoader(\n",
+    "        datasets.MNIST(dir, train=False,\n",
+    "                       transform=transforms.Compose([\n",
+    "                           transforms.ToTensor(),\n",
+    "                           transforms.Normalize((0.1307,), (0.3081,))\n",
+    "                       ])),\n",
+    "        batch_size=test_batch_size, shuffle=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now ready to train our network using Orca Pytorch Estimator:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "tags": [
+     "outputPrepend"
+    ]
+   },
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "...\n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:427 - [Epoch 5 58560/60160][Iteration 935][Wall Clock 85.759594242s] Trained 320.0 records in 0.083955904 seconds. Throughput is 3811.5247 records/second. Loss is 0.06152063. \n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:427 - [Epoch 5 58880/60160][Iteration 936][Wall Clock 85.835908327s] Trained 320.0 records in 0.076314085 seconds. Throughput is 4193.1973 records/second. Loss is 0.032923408. \n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:427 - [Epoch 5 59200/60160][Iteration 937][Wall Clock 85.913411948s] Trained 320.0 records in 0.077503621 seconds. Throughput is 4128.8394 records/second. Loss is 0.07016848. \n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:427 - [Epoch 5 59520/60160][Iteration 938][Wall Clock 85.992577896s] Trained 320.0 records in 0.079165948 seconds. Throughput is 4042.1418 records/second. Loss is 0.03392674. \n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:427 - [Epoch 5 59840/60160][Iteration 939][Wall Clock 86.071633508s] Trained 320.0 records in 0.079055612 seconds. Throughput is 4047.7834 records/second. Loss is 0.029882258. \n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:427 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Trained 320.0 records in 0.052754554 seconds. Throughput is 6065.827 records/second. Loss is 0.07037376. \n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:472 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Epoch finished. Wall clock time is 87921.741495 ms\n",
+      "2021-02-22 17:31:49 INFO  DistriOptimizer$:111 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Validate model...\n",
+      "[Stage 1893:>                                                       (0 + 1) / 1]\n",
+      "2021-02-22 17:31:51 INFO  DistriOptimizer$:177 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] validate model throughput is 5835.1914 records/second\n",
+      "2021-02-22 17:31:51 INFO  DistriOptimizer$:180 - [Epoch 5 60160/60160][Iteration 940][Wall Clock 86.124388062s] Top1Accuracy is Accuracy(correct: 9791, count: 10000, accuracy: 0.9791)\n"
```

Comment:
The validation accuracy is already displayed here. Is it still necessary to do `est.evaluate(data=test_loader, batch_size=test_batch_size)` ?

Reply:
We can put it here to demonstrate how to do evaluation.